qiskit_nature/second_q/drivers/pyscfd/pyscfdriver.py

```diff
@@ -295,7 +295,6 @@ def chkfile(self, chkfile: str) -> None:
         self._chkfile = chkfile
 
     @staticmethod
-    @_optionals.HAS_PYSCF.require_in_call
```

Comment:
Just to check: you are removing this because this is already checked via the `require_in_instance` decorator of the class?

Reply:
Yes, no need to check twice